index.html

```diff
@@ -511,44 +511,23 @@ <h4>
     <p>
       The NFC Forum has standardized a small set of useful sub record types in
       [[NFC-RTD]] (Resource Type Definition specifications) called
-      <dfn>well-known type</dfn>s, for instance text, URL, media and
-      opaque binary data. In addition, there are record types designed for more
+      <dfn>well-known type record</dfn>s, for instance text, URL, media and others.
+      In addition, there are record types designed for more
       complex interactions, such as smart posters (containing optional embedded
       records for url, text, signature and actions), and handover records.
     </p>
     <p>
-      These sub record types can be stored in the <dfn>well-known type record</dfn>.
+      The type information stored in the <a>TYPE field</a> of
+      <a>well-known type records</a> can be of two kinds: global or local type.
```

Comment:
Shall we link "global" to the section "Well-known global types" and "local type" to the section "Well-known local types"?

Reply:
fixed